ui/app/pages/create-account/connect-hardware/index.scss

```diff
@@ -132,6 +113,14 @@
     color: #9b9b9b;
     margin-top: 10px;
     margin-bottom: 15px;
+
+    &__link {
+      @include H6;
+
+      text-align: center;
+      color: #2f9ae0;
+      cursor: pointer;
```

Comment:
Is this necessary? :thinking: This is only being used on `<a>` tags as far as I can tell, which should already have a pointer cursor.

Reply:
Ah, I found an exception. It would still be better to use clickable elements and remove this rule though.

Reply:
I think this can safely be removed now? This is no longer used on non-pointer elements
```suggestion
```

Reply:
++